index.html

```diff
@@ -982,7 +982,17 @@ <h3>Contexts</h3>
 the context.
           </dd>
         </dl>
-
+        <p class="note">
+Though this specification requires that a <code>@context</code> <a>property</a>
+be present, there is no requirement that the value of the <code>@context</code>
+property be processed using JSON-LD. This is to support compatibility with
+certain JSON processing libraries such as those that may be used when the
```

Comment:
I dislike how this is phrased, the use of "certain" sounds weasely, like some JSON libraries are insufficient but others are good. Truth is, any plain JSON library isn't going to care about JSON-LD, but it can do a syntax check for specific values in a specific field. Suggest backing this off a bit:

```suggestion
This is to support processing using plain JSON libraries such as those...```

Reply:
changes made in [665e076](https://github.com/w3c/vc-data-model/pull/535/commits/665e07665cfb3bdfa4c5b625257e49bc709109fa)